python-sdk/hatchet_sdk/worker.py

```diff
@@ -2,49 +2,75 @@
 import json
 import signal
 import sys
-from threading import Thread, current_thread
 import threading
 import time
+from concurrent.futures import Future, ThreadPoolExecutor
+from threading import Thread, current_thread
+from typing import Any, Callable, Dict
 
 import grpc
-from typing import Any, Callable, Dict
-from .workflow import WorkflowMeta
-from .clients.dispatcher import GetActionListenerRequest, ActionListenerImpl, Action
-from .dispatcher_pb2 import ActionType, StepActionEvent, StepActionEventType, GroupKeyActionEvent, GroupKeyActionEventType, STEP_EVENT_TYPE_COMPLETED, STEP_EVENT_TYPE_STARTED, STEP_EVENT_TYPE_FAILED, GROUP_KEY_EVENT_TYPE_STARTED, GROUP_KEY_EVENT_TYPE_COMPLETED, GROUP_KEY_EVENT_TYPE_FAILED
-from .client import new_client
-from concurrent.futures import ThreadPoolExecutor, Future
 from google.protobuf.timestamp_pb2 import Timestamp
+
+from .client import new_client
+from .clients.dispatcher import (
+    Action,
+    ActionListenerImpl,
+    GetActionListenerRequest,
+)
 from .context import Context
+from .dispatcher_pb2 import (
+    GROUP_KEY_EVENT_TYPE_COMPLETED,
+    GROUP_KEY_EVENT_TYPE_FAILED,
+    GROUP_KEY_EVENT_TYPE_STARTED,
+    STEP_EVENT_TYPE_COMPLETED,
+    STEP_EVENT_TYPE_FAILED,
+    STEP_EVENT_TYPE_STARTED,
+    ActionType,
+    GroupKeyActionEvent,
+    GroupKeyActionEventType,
+    StepActionEvent,
+    StepActionEventType,
+)
 from .logger import logger
+from .workflow import WorkflowMeta
 
 
 class Worker:
-    def __init__(self, name: str, max_runs: int | None = None, debug=False, handle_kill=True):
+    def __init__(
+        self,
+        name: str,
+        max_runs: int | None = None,
+        debug=False,
+        handle_kill=True,
+    ):
         self.name = name
```

Comment:
We should be modifying the `self.name` to be namespaced as well, to match the TS SDK: https://github.com/hatchet-dev/hatchet-typescript/pull/27/files#diff-2bba1bd36b8922955278a73eeaca7a34816e0ea0a63c498f0f6cfc028010f40d

Reply:
Note that the `event_triggers` when creating the workflow should also be modified.